openassessment/templates/openassessmentblock/student_training/student_training.html

```diff
@@ -93,23 +92,23 @@ <h3 class="student-training__display__title">
                                 <li
                                     class="field field--radio is--required assessment__rubric__question has--options is--showing ui-slidable__container training__assessment__rubric__question--{{ criterion.order_num }}"
                                 >
-                                    <h4 class="question__title ui-slidable__control">
+                                    <div class="question__title ui-slidable__control">
                                         <button class="ui-slidable" aria-expanded="true" id="oa_training_{{ xblock_id }}_criterion--{{ criterion.order_num }}" aria-controls="oa_training_{{ xblock_id }}_content_criterion--{{  criterion.order_num }}">
                                             <span class="icon fa fa-caret-right" aria-hidden="true"></span>
                                             <span id="training__assessment__rubric__prompt--{{ criterion.order_num }}__{{ xblock_id }}" class="question__title__copy">{{ criterion.prompt }}</span>
                                             <span class="label--required sr">* ({% trans "Required" %})</span>
                                         </button>
-                                    </h4>
+                                    </div>
                                     <div class="ui-slidable__content" aria-labelledby="oa_training_{{ xblock_id }}_criterion--{{ criterion.order_num }}" id="oa_training_{{ xblock_id }}_content_criterion--{{  criterion.order_num }}">
                                         <div class="step__message message message--correct is--hidden">
-                                            <h3 class="message__title">{% trans "Selected Options Agree" %}</h3>
+                                            <h5 class="message__title">{% trans "Selected Options Agree" %}</h5>
```

Comment:
I was assuming that this is a child of the header defined above ("Training Assessment # {{ current_progress_num }} of {{ num_to_complete }}"). It appears that the form elements within are being displayed as peers instead of children, which seems a bit funky to me. Did @cptvitamin request this?

![image](https://cloud.githubusercontent.com/assets/484484/20901488/c9cb27a4-bb00-11e6-9be2-357e5489d518.png)